rpm/SPECS/docker-ce.spec

```
@@ -22,7 +22,10 @@ Requires: container-selinux >= 2:2.74
 Requires: libseccomp >= 2.3
 Requires: systemd
 Requires: iptables
+%if ! 0%{?rhel}
```

Should we make this conditional based on `rhel` (/centos) version? If it's not deprecated in rhel7 and centos7, (or 8) we could make it something like;

```suggestion
%if 0%{?rhel} == 7 || 0%{?rhel} == 8
```

---

Using straight (`==`) comparisons to account for other distros who would like not have the `rhel` macro (but double check if my suggestion is correct 😅)

---

Well, libcgroup is deprecated in RHEL 7 already: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/resource_management_guide/chap-using_libcgroup_tools

> The libcgroup package, which was the main tool for cgroup management in previous versions of Red Hat Enterprise Linux, is now deprecated. To avoid conflicts, do not use libcgroup tools for default resource controllers (listed in [Available Controllers in Red Hat Enterprise Linux 7](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/resource_management_guide/br-Resource_Controllers_in_Linux_Kernel#itemlist-Available_Controllers_in_Red_Hat_Enterprise_Linux_7)) that are now an exclusive domain of systemd. This leaves a limited space for applying libcgroup tools, use it only when you need to manage controllers not currently supported by systemd, such as net_prio.

---

The devil can be in the detail

> This leaves a limited space for applying libcgroup tools, use it only when you need to manage controllers not currently supported by systemd, such as net_prio.

so I think we should consider keeping it (for now) for the versions that still carry the package

---

Ok, then. I used the expression `%if 0%{?rhel} <= 8` does it fly for you ?